components/org.wso2.carbon.identity.oidc.session/src/main/java/org/wso2/carbon/identity/oidc/session/servlet/OIDCLogoutServlet.java

```diff
@@ -717,7 +741,8 @@ private void addAuthenticationRequestToRequest(HttpServletRequest request,
     }
 
     private void sendRequestToFramework(HttpServletRequest request, HttpServletResponse response, String sessionDataKey,
-                                        String type) throws ServletException, IOException {
+                                        String type)
+            throws ServletException, IOException {
```

Comment:
Can't we put this along with previous line?

Reply:
Addressed with https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1519/commits/b6b16b4e9375169a8e8109dead588f7651e3a54b.